### PR TITLE
User update profile

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- ticket redmine: https://edu-redmine.sun-asterisk.vn/issues/75154

## WHAT (optional)
- Add feature user update profile

## HOW
- Add feature user update profile

## Evidence (Screenshot or Video)
Screenshot from 2024-05-14 11-06-07
Screenshot from 2024-05-14 11-04-28
Screenshot from 2024-05-14 11-03-45

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: src/public/js/main.js [hunk 1]
```diff
@@ -1078,7 +1078,8 @@
       reader.readAsDataURL(files[0]);
 
       reader.onloadend = function(){ 
-        uploadFile.closest('.imgUp').find('.imagePreview').css('background-image', 'url('+this.result+')');
+        console.log(this.result);
```
Comment (huongnt-2545) [inline src/public/js/main.js]: xóa debug

File: src/test/user.test.ts [hunk 2]
```diff
@@ -350,3 +350,34 @@ describe('changePassword', () => {
     expect(checkPassword).toEqual(true);
   });
 });
+
+describe('updateProfile', () => {
```
Comment (huongnt-2545) [inline src/test/user.test.ts:354]: thêm các test case cho trường hợp update lỗi nhé

**Comment by ghost** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]

**Pull request merged**